<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>시작!</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txbLog">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>54</y>
     <width>370</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="pgbTask">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>264</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
